lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -220,19 +221,19 @@ export default function SubmitGradeForm({ student }) {
       {!!submitGradeError && (
         <ErrorDialog
           title="Submit Grade Error"
-          error={{ message: submitGradeError }}
+          error={ submitGradeError }
```

Comment:
While this code is being updated, can you update the `title` prop of this dialog and the one below so that it better fits how error descriptions appear elsewhere.

I would suggest something like "Unable to submit grade" and "Unable to fetch grade". Note the use of "Sentence case", rather than "Title Case" and the wording follows the form "Unable to <operation>" or "Failed to <operation>" rather than "<operation> error".